### PR TITLE
fix: Bug fixed - Added placeholder in search bar in apps section

### LINKED ISSUE #11927: Placeholder in search field in "apps" section
Found a bug? Please fill out the sections below. 👍

### Issue Summary

A summary of the issue. This needs to be a clear detailed-rich summary.

### Steps to Reproduce

1. Login or Signup to cal.com
2. From sidebar select "Apps"
3. Then in above right side, you'll see a text field.

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

### Actual Results

- No placeholder is there, user might not get what to do there.

### Expected Results

- Embed a placeholder in text field

### Technical details

- Browser version, screen recording, console logs, network requests: You can make a recording with [Bird Eats Bug](https://birdeatsbug.com/).
- Node.js version - v18.18.0
- Anything else that you think could be an issue.

Screenshot (108)

### PR DESCRIPTION
**Pull request by kilbha**
Added place holder in search bar in apps section.
Fixed: #11927

### CODE REVIEW

**Comment by PeerRich** [comment]
can you uncommit your yarn.lock? https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md#guidelines-for-committing-yarn-lockfile

**Comment by PeerRich** [comment]
not needed imo

**Comment by kilbha** [comment]
Thanks PeerRich

File: apps/web/pages/apps/index.tsx [hunk 1]
```diff
@@ -54,6 +54,7 @@ function AppsSearch({
       type="search"
       autoComplete="false"
       onChange={onChange}
+      placeholder="Search apps"
```
Comment (PeerRich) [inline apps/web/pages/apps/index.tsx:57]: missing i18n, use t("search_apps")

**Comment by PeerRich** [comment]
added correctly here https://github.com/calcom/cal.com/pull/11944

**Pull request closed without merging**